python/tvm/driver/tvmc/frontends.py

```diff
@@ -76,20 +78,19 @@ def load(self, path, shape_dict=None, **kwargs):
         """
 
 
-def import_keras():
-    """Lazy import function for Keras"""
-    # Keras writes the message "Using TensorFlow backend." to stderr
-    # Redirect stderr during the import to disable this
-    stderr = sys.stderr
-    sys.stderr = open(os.devnull, "w")
+def lazy_import(pkg_name, from_pkg_name=None, hide_stderr=False):
+    """Lazy import a frontend package or subpackage"""
     try:
-        # pylint: disable=C0415
-        import tensorflow as tf
-        from tensorflow import keras
+        if hide_stderr:
+            stderr = sys.stderr
+            sys.stderr = open(os.devnull, "w")
 
-        return tf, keras
+        return importlib.import_module(pkg_name, package=from_pkg_name)
+    except ImportError:
+        raise TVMCImportError({pkg_name})
```

Comment:
To avoid Python complaining about nested exceptions you have to explicitly state it, and I don't think this needs squiggles?
```suggestion
    except ImportError as error:
        raise TVMCImportError(pkg_name) from error
```